docker-entrypoint

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 set -e
+set -a
```

Comment:
I don't really understand the reason for setting this. I doubt scripts being sourced would always want to export all the internal variables they define. It feels cleaner to let scripts explicitly decide which ones they want to export.
(or each script could call `set -a` if they wanted)